docs/guides/TerraToolkitUpgradeGuide-v5.1.0-part2.md

```diff
@@ -0,0 +1,230 @@
+    '@babel/preset-react',
+  ];
+  const plugins = [
+    '@babel/plugin-transform-object-assign',
+    '@babel/plugin-proposal-object-rest-spread',
+  ];
+
+  return {
+    presets,
+    plugins,
+  };
+};
+```
+
+Be sure to check your project's current `.babelrc` config to ensure these presets and plugins are correct.
+
+### 4. Remove the `.babelrc` file
+The `.babelrc` is no longer needed with the addition of the `babel.config.js` file.
+
+### 5. Check Jest Versions
+If your project uses `jest` and/or `babel-jest` for unit testing, these dev dependencies will also need updated:
+
+```bash
+> npm update jest babel-jest
+```
+
+## Step 3. Add Browserslist Configuration
+To ensure all tools are properly configured (especially webpack since it generates your site assets!) you **must** provide a browserslist configuration in your project.
+
+[Browserslist](https://github.com/browserslist/browserslist) is a tool used by most of the build tools in your project (babel, eslint, various webpack plugins) to ensure you are developing correctly for the list of target browsers you provide.
+
+<details>
+<summary>
+Terra recommends using the list provided by browserslist-config-terra.
+</summary>
+
+See the [`browserslist-config-terra` documentation](https://github.com/cerner/browserslist-config-terra) for more info.
+
+### 1. Add `browserslist-config-terra` dev dependency
+```bash
+> npm install --save-dev browserslist-config-terra
+```
+### 2. Add a `browserslist` key to the `package.json`
+
+```json
+"browserslist": [
+  "extends browserslist-config-terra"
+]
+```
+</details>
+
+## Step 4. Check Aggregate Translations Usage
+If your project runs aggregate-translations for jest unit testing, you will need to remove the terra-toolkit aggregate-translations reference, since it no longer exists.
+
+### 1. Install terra-aggregate-translations
+```
+> npm install --save-dev terra-aggregate-translations
+```
+### 2. Update aggregate-translation imports
+```diff
+- const aggregateTranslations = require('terra-toolkit/scripts/aggregate-translations/aggregate-translations');
++ const aggregateTranslations = require('terra-aggregate-translations');
+```
+
+## Step 5. Update the Start Scripts 
+
+The start scripts in the `package.json` need to be updated. `tt-serve` is now a thin abstraction on [`webpack-dev-server`](https://webpack.js.org/configuration/dev-server/) and `tt-start-static` will only serve compiled site assets.
+
+### 1. Update the `start` script
+The `webpack-dev-server` cannot automatically load our defaults. Thus, a webpack.config must be provided at the root level or be passed via the `--config` flag in the package.json script. 
+
+```diff
+-  "start": "tt-serve",
++  "start": "tt-serve --config node_modules/terra-dev-site/config/webpack/webpack.config.js",
+```
+
+### 2. Add a `start-prod` script
+Add the following script to run `webpack-dev-server` with hot-reloading disabled to view the assets that are used during the test run.
```

Comment:
Could be more specific about what "he assets that are used during the test run" means